src/cephadm/cephadm.py

```diff
@@ -257,6 +297,10 @@ def get_sysctl_settings() -> List[str]:
     def firewall_service_name(self) -> str:
         return 'ceph'
 
+    @property
+    def osd_fsid(self) -> Optional[str]:
```

Comment:
Forgot https://github.com/ceph/cephadm-adoption-corpus existed

Reply:
I pushed a fix for this. I also added a long line to be emitted when the OSD is constructed but no osd_fsid value is set. This is because it's not clear to me when it is needed or not. If it ends up being useful for debugging we can leave it in but if it ends up just being noise, we can drop it in the future.